jdat_notebooks/cube_fitting/requirements

```diff
@@ -1,2 +1,3 @@
 astropy==1.3
 lmfit==0.9.13
+dust-extinction=0.8
```

Comment:
You'll want to use the comparison operator instead of the assignment operator here.

```suggestion
dust-extinction==0.8
```

Reply:
Done.

Reply:
Still not passing CI-check.

Reply:
Should it be 'dust-extinction' or 'dust_extinction' in requirements?